src/include/parser/expression/parsed_case_expression.h

```diff
@@ -24,6 +29,14 @@ class ParsedCaseExpression : public ParsedExpression {
     explicit ParsedCaseExpression(std::string raw)
         : ParsedExpression{common::CASE_ELSE, std::move(raw)} {};
 
+    ParsedCaseExpression(common::ExpressionType type, std::string alias, std::string rawName,
```

Comment:
You might want to do this with a copy constructor, i.e. `ParsedCaseExpression(const ParsedCaseExpression& other)`

and copy can be implemented as 
```
return std::make_unique<XXX>(*this);
```

Up to u, if you decide to change, change across all parsed expression